docs/en/operations/system_tables.md

```diff
@@ -742,17 +742,22 @@ WHERE
 
 If this query doesn't return anything, it means that everything is fine.
 
-## system.settings
+## system.settings {#system-tables-system-settings}
 
-Contains information about settings that are currently in use.
-I.e. used for executing the query you are using to read from the system.settings table.
+Contains information about settings that are currently in use. 
 
 Columns:
 
 - `name` (String) — Setting name.
-- `value` (String)  — Setting value.
+- `value` (String) — Setting value.
 - `changed` (UInt8) — Whether the setting was explicitly defined in the config or explicitly changed.
-
+- `description` (String) — General information about the setting. 
+- `min` (String) — Minimum value of the setting. Also accepts `Null`. 
+- `max` (String) — Maximum value of the setting. Also accepts `Null`.
+- `readonly` (UInt8) — Setting can't be changed by the user:
```

Comment:
It can be 0 or 1. In the sources it has the boolean type. It only shows whether the current user can change the setting. The further words doesn't needed.